lms/static/sass/multicourse/_home.scss

```diff
@@ -50,26 +59,77 @@
       }
 
       > hgroup {
+        @include left(0);
         @include box-sizing(border-box);
+        @include transition(all 0.2s linear 0s);
+        @include text-align(left);
         display: inline-block;
-        left: 0px;
+        top: 0px;
```

Comment:
When dealing with lengths, a 0 value should never ever have a unit.
https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS